### PR TITLE
chore: run shake periodically, and post if something goes wrong

### PR DESCRIPTION
**Pull request by kim-em**
As with #12004 I propose we merge this and try it out before actually removing `shake` from every CI run.

### CODE REVIEW

File: .github/workflows/shake.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        run: |
+          git config user.name "github-actions"
+          git config user.email "github-actions@github.com"
+
+      - name: Commit local changes and create PR
+        if: ${{ steps.git-check.outputs.changes == 'true' }}
+        run: |
+          DATE=$(date +%Y-%m-%d)
+          BRANCH="shake-$DATE"
+          git checkout -b "$BRANCH"
```
Comment (adomani) [inline .github/workflows/shake.yml:44]:
```suggestion
          git checkout -b "$BRANCH"  || exit 1
```
I do not know if this is needed, but I wonder what happens if the branch creation goes wrong -- probably not much anyway.

**Review by adomani — APPROVED** [review]
For what I can understand of reading the CI steps (which is not much), this looks good to me.  It would be great if we could test this, but I think that this can only happen once it is merged, right?

**Comment by semorrison** [comment]
I think to test this properly we're going to need to remove the `shake` step from standard CI, so we can exercise the cron job.

**Comment by YaelDillies** [comment]
> we're going to need to remove the `shake` step from standard CI, so we can exercise the cron job.

Should this PR do that as well, then?

**Comment by semorrison** [comment]
I no longer think this PR is a good idea, so closing.

**Pull request closed without merging**